package.json

```diff
@@ -51,5 +51,8 @@
     "metasync": "^0.3.31",
     "metatests": "^0.3.0",
     "prettier": "1.16.x"
+  },
+  "dependencies": {
+    "@metarhia/doc": "^0.1.0"
```

Comment:
Any reason why this is not a `devDependency`? Also, you should remove `metaschema` from `devDependencies` since it provides the same functionality.